swh/voters.py

```
@@ -0,0 +1,151 @@
+
+    total_rows = 0
+    first_chunk = True
+
+    for chunk in pd.read_csv(filepath, chunksize=chunk_size):
+        # Drop rows without valid coordinates
+        chunk = chunk.dropna(subset=[longitude_col, latitude_col])
+
+        # Create geometry
+        geometry = [
+            Point(lon, lat)
+            for lon, lat in zip(chunk[longitude_col], chunk[latitude_col])
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)